browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -47,7 +47,7 @@ constexpr const char* kAllowedUrlPrefixes[] = {
     // brave A/B testing
     "https://variations.brave.com/seed",
 
-    // Brave Today (production)
+    // Brave News (production)
     "https://brave-today-cdn.brave.com/",
```

Comment:
you would need to add new domains if there are any

Reply:
There aren't any :smile: